src/Tabs/Tabs.js

```diff
@@ -233,11 +239,13 @@ class Tabs extends Component {
 
     return (
       <div style={prepareStyles(Object.assign({}, style))} {...other}>
-        <div style={prepareStyles(Object.assign(styles.tabItemContainer, tabItemContainerStyle))}>
-          {tabs}
-        </div>
-        <div style={{width: inkBarContainerWidth}}>
-          {inkBar}
+        <div className={labelsContainerClassName}>
```

Comment:
Oh, thanks for providing more detail. I'm going to reject this change proposal.

The `<Tab />` accepts a child component for convinience. That works for the simple use cases, but not for the complexe one. For instance, what if you don't want to render the hidden tabs? I would rather decoupled the tabs component to the tab content components. They are mixed right now. Have decoupled them on the `next` branch. That's simpler and more flexible.